tests/draft2019-09/refRemote.json

```diff
@@ -76,7 +76,7 @@
         "schema": {
             "$id": "http://localhost:1234/scope_change_defs1.json",
             "type" : "object",
-            "properties": {"list": {"$ref": "#/$defs/baz"}},
+            "properties": {"list": {"$ref": "folder/"}},
```

Comment:
Similar to the `$defs` change above, this change is because an implementation isn't required to index non-canonical schema URI's (per 8.2.2.2).